cmd/csender/main.go

```diff
@@ -0,0 +1,157 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/cloudradar-monitoring/cagent/pkg/csender"
+)
+
+var (
+	// set on build:
+	// go build -o csender -ldflags="-X main.version=$(git --git-dir=src/github.com/cloudradar-monitoring/cagent/.git describe --always --long --dirty --tag)" github.com/cloudradar-monitoring/cangent/cmd/csender
+	version string
+)
+
+type boolFlag struct {
+	set   bool
+	value bool
+}
+
+func (sf *boolFlag) Set(value string) error {
+	i, err := strconv.Atoi(value)
+	if err != nil {
+		return err
+	}
+
+	switch i {
+	case 0:
+		sf.value = false
+	case 1:
+		sf.value = true
+	default:
+		return fmt.Errorf("arg can be either 0 or 1")
+	}
+
+	sf.set = true
+	return nil
+}
+
+func (sf *boolFlag) String() string {
+	var s string
+	if sf.set {
+		if sf.value {
+			s = "1"
+		} else {
+			s = "0"
+		}
+	}
+
+	return s
+}
+
+func (sf *boolFlag) BoolPtr() *bool {
+	if !sf.set {
+		return nil
+	}
+
+	return &sf.value
+}
+
+func fatal(msg string) {
+	_, _ = fmt.Fprintln(os.Stderr, msg)
```

Comment:
cagent main.go uses logrus.Fatalf for logging fatal messages. Probably we should use it here for consistency.

Reply:
Yes, I know. But csender is super slim and doesn't need any logging mechanism at all – cause it runs once and exits with message/status. I just didn't want to add an unnecessary dependency